client/my-sites/site-settings/custom-content-types/index.jsx

```diff
@@ -116,10 +95,45 @@ class CustomContentTypes extends Component {
 		);
 	}
 
-	renderContentTypeSettings( fieldName, fieldLabel, fieldDescription ) {
+	renderContentTypeSettings( name, label, description ) {
+		const {
+			activatingCustomContentTypesModule,
+			fields,
+			handleAutosavingToggle
+		} = this.props;
+		return (
+			<div className="custom-content-types__module-settings">
+				{
+					name !== 'post'
+						? (
+							<CompactFormToggle
+								checked={ !! fields[ name ] }
+								disabled={ this.isFormPending() || activatingCustomContentTypesModule }
+								onChange={ handleAutosavingToggle( name ) }
+							>
+								{ label }
+							</CompactFormToggle>
+						)
+						: label
+				}
+
+				{ this.renderPostsPerPageField( name, label ) }
+
+				<FormSettingExplanation isIndented>
+					{ description }
+				</FormSettingExplanation>
+			</div>
+		);
+	}
+
+	renderBlogPostSettings() {
+		const { translate } = this.props;
+		const fieldLabel = translate( 'Blog posts' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **57** times:
`translate( 'Blog Posts' )` *ES Score: 10*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).